File: exercises_3/python/solutions/exampleSolutionPart1.py
import skimage.io
import numpy as np
import utils
import matplotlib.pyplot as plt

img1 = skimage.io.imread('cine_MR_1.png')
img2 = skimage.io.imread('cine_MR_2.png')

img1 = (np.flipud(np.double(img1))).T
img2 = (np.flipud(np.double(img2))).T


plt.figure(1)
utils.dispImage(img1)
plt.figure(2)
utils.dispImage(img2)
plt.figure(3)
utils.dispColourOverlayImage(img1, img2)

File: exercises_3/matlab/solutions/exampleSolutionPart1.m
clear;
close all;

%load lung MR images
cine_MR_1 = imread('cine_MR_1.png');
cine_MR_2 = imread('cine_MR_2.png');

%convert to double and change to 'standard orientation'
cine_MR_1 = flip(double(cine_MR_1'), 2);
cine_MR_2 = flip(double(cine_MR_2'), 2);

%display images
figure
dispImage(cine_MR_1);
figure
dispImage(cine_MR_2);
figure
dispColourOverlayImage(cine_MR_1, cine_MR_2);